### PR TITLE
Enable CORS in local provider

### PR DESCRIPTION
**Pull request by alvaroloes**
## Description
Using the local provider from a web browser was giving CORS errors. This PR fixes that

## Checks
- [x] Project Builds
- [x] Project passes tests and checks
- ~[ ] Updated documentation accordingly~

### CODE REVIEW

**Review by NickSeagull — APPROVED** [review]
💣

File: packages/framework-provider-local-infrastructure/src/index.ts [hunk 1]
```diff
@@ -40,6 +41,12 @@ export const Infrastructure = {
     const graphQLService = new GraphQLService(userProject)
     router.use('/graphql', new GraphQLController(graphQLService).router)
     expressServer.use(express.json())
+    expressServer.use(cors())
+    expressServer.use(function (req, res, next) {
+      res.header('Access-Control-Allow-Origin', '*')
```
Comment (juanjoman) [inline packages/framework-provider-local-infrastructure/src/index.ts:46]:
```suggestion
      res.header('Access-Control-Allow-Origin', '127.0.0.1')
```
Wouldn't it be better to just allow your machine to do this?

**Review by juanjoman — APPROVED** [review]
Amazing! Just left a comment/suggestion

File: packages/framework-provider-local-infrastructure/src/index.ts (hunk 1 above)
Comment (alvaroloes) [reply packages/framework-provider-local-infrastructure/src/index.ts:46]: Never thought of this! But one question: If I write `'*'`, Can another machine access it? I mean, that other machine would be in another network and the developer should allow incoming requests through a specific port...

Comment (juanjoman) [reply packages/framework-provider-local-infrastructure/src/index.ts:46]: Only if you port forward the application on local (which is not done by default) or if you expose a container with the local env on a docker for example (as a normal application, exposed to the internet). It's not a super big deal, but something nitpicky haha

Comment (alvaroloes) [reply packages/framework-provider-local-infrastructure/src/index.ts:46]: Ok, I'll change it and we will see if in the future we need some configuration about this

Comment (alvaroloes) [reply packages/framework-provider-local-infrastructure/src/index.ts:46]: Eventually this didn't worked and I went back to to the `*`

**Pull request merged**